src/member-api.js

```diff
@@ -54,42 +75,148 @@ export class MemberApi extends TypedEmitter {
    * @param {string} [opts.roleName]
    * @param {string} [opts.roleDescription]
    * @param {number} [opts.timeout]
-   *
-   * @returns {Promise<import('./generated/rpc.js').InviteResponse_Decision>}
+   * @returns {Promise<(
+   *   typeof InviteResponse_Decision.ACCEPT |
+   *   typeof InviteResponse_Decision.REJECT |
+   *   typeof InviteResponse_Decision.ALREADY
+   * )>}
    */
-  async invite(deviceId, { roleId, roleName, roleDescription, timeout }) {
+  async invite(
```

Comment:
Not for this PR, but a note for the future: I don't think it's a great API design having these long-running promises. It means we need special handling on the RPC bridge, because normally we would consider a promise taking this long to resolve an error. At some point I think we should change this to an events-based API e.g. `on('invite-accepted', (inviteId) => {})`